packages/origin-247-certificate/src/certificate.service.ts

```diff
@@ -0,0 +1,103 @@
+import { GetAllCertificatesQuery, GetCertificateQuery } from '@energyweb/issuer-api';
+import { Injectable } from '@nestjs/common';
+import { QueryBus } from '@nestjs/cqrs';
+import { InjectQueue } from '@nestjs/bull';
+import { IClaim } from '@energyweb/issuer';
+import { Queue } from 'bull';
+import {
+    ICertificate,
+    IClaimCommand,
+    IIssueCommand,
+    ITransferCommand,
+    ISuccessResponse,
+    BlockchainActionType,
+    BlockchainAction,
+    IIssuedCertificate,
+    IIssueCommandParams
+} from './types';
+
+const jobOptions = {
```

Comment:
Shouldn't they be taken from env?

Reply:
No, not really in this case, this service is the only entity responsible for this queue